WordPress/src/main/java/org/wordpress/android/ui/people/PeopleInviteFragment.java

```diff
@@ -442,6 +454,10 @@ private void checkAndSend() {
             return;
         }
 
+        //set the  "SEND" option disabled
+        mInviteOperationInProgress = true;
+        getActivity().invalidateOptionsMenu();
```

Comment:
This couple of lines (setting the boolean and then calling `invalidateOptionsMenu()`) seem to appear a few times so, may I suggest refactoring those into a function? Thanks!